### ISSUE TITLE
newer API version - 4.4

### ISSUE THREAD

**Issue by furas** [DanielJDufour/hatebase#3]
Code needs 

    version = '4-4'

to work with newest API.

**Comment by DanielJDufour** [comment]
Hi, @furas .  Thank you for catching this!  Would you be interested in submitting a pull request?  Thank you for your consideration.

**Comment by furas** [comment]
I will not submit a pull request because I didn't clone repo to make modification. 

I only tried to answer for question on Stackoverflow [Error in downloading data using API (python)](https://stackoverflow.com/questions/68402237/error-in-downloading-data-using-api-python)  
and digging in source code I found that it needs `version = '4-4'` to work with newest API.

---

But later I figure out that it can be resolved also with argument 

```
hatebase = HatebaseAPI({"key": key, "version": "4-4"})
```

but without documentation other people will not know it.
